ui/src/components/WorkerTypesTable/index.jsx

```diff
@@ -85,18 +73,10 @@ export default class WorkerTypesTable extends Component {
   };
 
   createSortedWorkerTypesConnection = memoize(
-    (
-      workerTypesConnection,
-      awsProvisionerWorkerTypeSummaries,
-      sortBy,
-      sortDirection
-    ) => {
+    (workerTypesConnection, sortBy, sortDirection) => {
       const sortByProperty = camelCase(sortBy);
       // Normalize worker types for aws-provisioner-v1
-      const workerTypes = normalizeWorkerTypes(
-        workerTypesConnection,
-        awsProvisionerWorkerTypeSummaries
-      );
+      const workerTypes = normalizeWorkerTypes(workerTypesConnection);
```

Comment:
We no longer need to `normalizeWorkerTypes`. We can just use `workerTypesConnection` directly.